core/src/main/scala/org/broadinstitute/dsde/rawls/policy/PolicyService.scala

```diff
@@ -46,4 +48,16 @@ class PolicyService(tpsDAO: TpsDAO)(implicit val ec: ExecutionContext) {
 
     tpsDAO.createPao(req, ctx)
   }
+
+  /**
+    * The documentation for this TPS API is misleading. It will merge the target PAO into the source
+    * PAO, NOT the other way around. While it might be confusing to see the destination workspace id
+    * used as the `sourceObjectId`, this is the correct way to call the TPS API.
+    */
```

Comment:
In the `TpsApiController` in TPS, `mergeFromPao` is called with the destination and source object ids swapped around ([source](https://broadworkbench.atlassian.net/browse/CORE-438)). We may want to fix this, but we would need to be careful rolling out the change now that upstream services are dependent on it being implemented this way.